src/videojs-hls.js

```diff
@@ -585,7 +621,8 @@ videojs.HlsHandler.prototype.selectPlaylist = function () {
     effectiveBitrate,
     sortedPlaylists = this.playlists.master.playlists.slice(),
     bandwidthPlaylists = [],
-    i = sortedPlaylists.length,
+    now = +new Date(),
```

Comment:
Oops, forgot to address this one in my update. We should probably do a sweep to fix this all over the place.